Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -521,6 +522,23 @@ bool ReflectometryReductionOneAuto::checkGroups() {
   return false;
 }
 
+Mantid::API::Workspace_sptr
+ReflectometryReductionOneAuto::handleTransmissionGroup(
+    Mantid::API::WorkspaceGroup_sptr transG) {
```

Comment:
I would name this function something else. As a general rule avoid `manage` `utility` `helper` or `handler` from names in software if possible.